src/plugins/specload/specload.c

```diff
@@ -41,8 +41,6 @@ struct change
 // TODO: allow more changes
 static struct change allowedChanges[] = { { "description", true, true, true },
 					  { "opt/help", true, true, true },
-					  { "default", true, true, false },
```

Comment:
Other pure documentary meta-data is: order, comment, see, rationale, requirement and example.

Reply:
I did not add `comment/#` (only `comment`) or `see/#`, because there is no mechanism for array-style metadata in `specload` right now.